frameworks/hdfs/tests/config.py

```diff
@@ -74,7 +75,23 @@ def run_hdfs_command(svc_name, command):
     return rc, output
 
 
-def check_healthy(count=DEFAULT_TASK_COUNT):
-    sdk_plan.wait_for_completed_deployment(PACKAGE_NAME, timeout_seconds=25 * 60)
-    sdk_plan.wait_for_completed_recovery(PACKAGE_NAME, timeout_seconds=25 * 60)
-    sdk_tasks.check_running(PACKAGE_NAME, count)
+def check_healthy(service_name=PACKAGE_NAME, count=DEFAULT_TASK_COUNT, recovery_expected=False):
```

Comment:
I think I would argue for making `service_name` a non-default parameter. This would remove the chance of forgetting to pass in the correct value (generally `FOLDERED_SERVICE_NAME`).